### ISSUE TITLE
Bluish cystic swelling below medial canthus

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10061]
**Preferred term label:**
Bluish cystic swelling below medial canthus

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
This is from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5848278/ and is admittedly a bundled term so it would be fine if more than one code were used to specify it.

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
See  [Dacryocystocele HP:0030752](https://hpo.jax.org/app/browse/term/HP:0030752)
